var-spacing
===========

This rules validates the correct alignment of variable declarations:

__wrong:__

``` javascript
let foo = {},
    foobar = [];
```

__right:__

``` javascript
let foo    = {},
    foobar = [];
```
